backend/functions/src/matching/functions.js

```diff
@@ -119,7 +121,12 @@ async function makeMatches(courseId, groupSize = 3) {
           .doc(courseId)
           .collection('groups')
           .doc(group.groupNumber.toString())
-          .set(group, { merge: true })
+          .set(
+            group,
+            { merge: true },
+            { createTime: admin.firestore.FieldValue.serverTimestamp() },
+            { updateTime: admin.firestore.FieldValue.serverTimestamp() }
```

Comment:
I don't think these two lines are necessary, I commented them out and it still works. Since this is JavaScript it probably ignores extra arguments passed to the `.set()` function instead of erroring